src/content/architecture/components/lighthouse/_index.en.md

```diff
@@ -11,20 +11,17 @@ The below digram shows the basic Lighthouse architecture.
 
 ![Lighthouse Architecture](/images/lighthouse/architecture.png)
 
-### Lighthouse Controller
-This is the central discovery controller that gathers the information from the clusters, decides what information is to be shared and distributes the information as newly defined CRDs (Kubernetes custom resources).
-
-The Lighthouse controller uses [kubefed](https://github.com/kubernetes-sigs/kubefed) to discover clusters and to extract and distribute aggregated Service information used to perform DNS resolution.
+### Lighthouse Agent
+The Lighthouse Agent runs in every cluster and accesses the Kubernetes API server running in the broker cluster to exchange service metadata information with other clusters. Local service information is exported to the broker and service information from other clusters is imported.
 
 #### WorkFlow
-The workflow is as follows.
+The workflow is as follows:
 
-- Lighthouse controller registers to be notified when clusters join and unjoin the kubefed control plane.
-- When notified about a join event, it retrieves the credentials from kubefed and registers a watch for Service creation and removal on that cluster.
-- When notified of a new Service created, it creates a MultiClusterService resource with the Service info and distributes it to all the clusters that are connected to the kubefed  control plane.
-- When notified of a Service deleted, its info is removed from the MultiClusterService resource and re-distributed.
+- Lighthouse agent connects to the broker's K8s API server.
+-  For every service in the local cluster, it creates a corresponding MultiClusterService resource and exports it to the broker to be consumed by other clusters.
+- For every MultiClusterService resource in the broker exported from another cluster, it creates a copy of it in the local cluster.
 
-![Lighthouse Controller WorkFlow](/images/lighthouse/controllerWorkFlow.png)
+![Lighthouse Agent WorkFlow](/images/lighthouse/controllerWorkFlow.png)
```

Comment:
I think you removed this file in favor of "architecture.png".

Reply:
This file was updated not removed. Since there is a size reduction it is shown in red.

Reply:
Ah okay, I misunderstood the UI. 👍